Customizing/global/plugins/Services/Cron/CronHook/UserOrguImport/classes/Data/SpoutExtractor.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace CaT\IliasUserOrguImport\Data;
+
+use Box\Spout\Reader\AbstractReader;
+use Box\Spout\Reader\ReaderFactory;
+use Box\Spout\Writer\WriterFactory;
+use Box\Spout\Common\Type;
+
+abstract class SpoutExtractor implements DataExtractor
+{
+	abstract protected function type();
+	abstract protected function setOptions(AbstractReader $reader);
```

Comment:
If you would use an `abstract protected function reader();` instead of these two here, you would spare that odd `setOptions` in the `SpoutCSVExtractor`.